src/tests/roc_audio/test_resampler.cpp

```diff
@@ -32,7 +32,8 @@ enum {
     FrameSize = 512,
 
     OutSamples = FrameSize * 100 + 1,
-    InSamples = OutSamples + (FrameSize * 3)
+    InSamples = OutSamples + (FrameSize * 3),
+    SampleRate = 44100
```

Comment:
It seems that SampleRate is unused.

Reply:
removed SampleRate.

Reply:
Now we have unused DefaultFrameLength :)